### DISCUSSION TITLE
how to get back the subtitle background after the update to v 0.39?

### DISCUSSION THREAD

**Discussion by bheeshmpita** [mpv-player/mpv#14937] [Q&A] [answered]
I have lost the background in spite of same config file after the update. Please guide me how to revert.

**Comment by llyyr** [comment]
--sub-back-color? I'm not sure what you mean specifically, mpv should log some error if you're using an option that was removed or changed.

**Comment by natural-harmonia-gropius** [comment, marked as answer]
```
sub-outline-size=0
sub-border-style=background-box
sub-shadow-offset=4
sub-back-color=0.0/0.0/0.0/0.75
```

**Reply by bheeshmpita** [reply]
adding `sub-border-style=background-box` got it working again. Thanks for the help.

**Reply by llyyr** [reply]
that doesn't make any sense. sub-border-style is a new option. How was it working before???

**Reply by natural-harmonia-gropius** [reply]
> How was it working before???

https://github.com/natural-harmonia-gropius/mpv-config/commit/7ebb3903363941c48f37e7452b3fd0c31ce49711
